README.md

```diff
@@ -55,6 +55,21 @@ generating subscription for profile nginx-catalog/weaveworks-nginx:
 
 Then the result will be in profile-subscription.yaml file.
 
+### Prepare
```

Comment:
don't forget to run `mdtoc -inplace README.md` to update the TOC (i left instructions for what to download and run at the top of the readme)

Reply:
Good call! :) I would totally have forgotten it.

Reply:
worth adding that to a `make` something or other? 🤔

Reply:
actually yeh, i will do that in another pr real quick: makefile needs a general tidy

Reply:
I think so, yes. Anything that has to be run by hand will eventually be forgotten at least once, right? :) So making it as easy as possible is a good point.

I can add that in my PR?

Reply:
Sounds good! :)